extensions/f1-standings/src/constructor.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import { ActionPanel, Action, Icon, List, Color } from "@raycast/api";
+import axios from "axios";
```

Comment:
A question from a non-js dev: was it possible to use `fetch` API to avoid depending on a third-party library?

Reply:
Also `fetch` has to be installed as a dependency in node: `node-fetch`. I think Node 18 will include "fetch".
But I replaced `axios` with `node-fetch` now. No problem for me if that is the preferred way in Raycast.

Reply:
There is no preferred way, axios can be used 😄